vnet/lib/ext/celluloid.rb

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+
+require 'celluloid/proxies/abstract_proxy'
+
+module Celluloid
+  class AbstractProxy
+    def tap
+      raise "OpenVNet has disabled tap method for all Celluloid proxies."
+    end
+
+    def nil?
+      raise "OpenVNet has disabled nil? method for all Celluloid proxies."
+    end
```

Comment:
What is the reason for disabling these? Perhaps a comment is in order here?

Reply:
When calling tap or nil? on an actor proxy object it sends the command to be executed by the actor.

So instead of the expected behavior of checking if the local variable is nil or passing the actor proxy object through tap locally, it sends the code block to be run remotely.

I'll add comments.